sdk/helper/tokenhelper/tokenhelper.go

```diff
@@ -38,6 +38,9 @@ type TokenParams struct {
 
 	// The TTL to user for the token
 	TokenTTL time.Duration `json:"token_ttl" mapstructure:"token_ttl"`
+
+	// The maximum number of times a token issued from this role may be used.
+	TokenMaxNumUses int `json:"token_max_num_uses" mapstructure:"token_max_num_uses"`
```

Comment:
Should we use `omitempty` here? Assuming this value gets output to the user at some point, I wonder if `0` would be confusing. Internally zero means "unlimited" but that's mostly an implementation detail, yeah?